<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighlighterSettingsPage</class>
 <widget class="QWidget" name="HighlighterSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="definitionFilesGroupBox">
     <property name="title">
      <string>Syntax Definition Files</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="Utils::PathChooser" name="definitionFilesPath" native="true">
          <zorder>locationLabel</zorder>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="resetButton">
          <property name="toolTip">
           <string>Reset to default</string>
          </property>
          <property name="text">
           <string>R</string>
          </property>
          <property name="icon">
           <iconset resource="../../coreplugin/core.qrc">
            <normaloff>:/core/images/reset.png</normaloff>:/core/images/reset.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>locationLabel</zorder>
     <zorder>definitionFilesPath</zorder>
     <zorder>resetButton</zorder>
     <zorder>resetButton</zorder>
     <zorder>resetButton</zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>146</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../coreplugin/core.qrc"/>
 </resources>
 <connections/>
</ui>
